src/MFPEncoder.ts

```diff
@@ -37,6 +47,7 @@ export interface EncodeMapOptions {
 export default class MFPBaseEncoder {
   readonly url: string;
   private scratchCanvas: HTMLCanvasElement = document.createElement('canvas');
+  private printResolution = 96;
```

Comment:
You should not keep it at the class level but instead pass it around where needed. Like it was done actually.
We should be able to call any encoder method and it would work, without strange magics.
With your change, you cannot call `encodeMVTLayerState` without having first call `encodeMap`.

Reply:
_strange magic_ ? everything in IT are magic in this case.

Anyway. I was juts thinking that passing this argument in every method is not adapted.
IMO, we could add a "setPrintResolution" method, or use the customizer. Or at least pass the whole options through the methods.

Reply:
The default value is strange to me...
96 looks like a DPI, but the variable name is resolution, and it looks like to be used as a resolution...

Reply:
sbrunner, you're right. Default is 1 now, as it's not really possible to define a "default value"

I've now added a setPrintResolution methode and removed it from the encode options. That's a good compromise I think.

Reply:
@ger-benjamin I have discussed the topic with @sbrunner: we really think passing the `printResolution` explicitely as parameter makes the lib easier to discover and easier to use correctly.

Reply:
That's not part of the aim of this PR. I've revert this change.